python/ql/src/experimental/Security/CWE-022/ZipSlip.ql

```diff
@@ -18,5 +18,6 @@ import DataFlow::PathGraph
 
 from ZipSlipConfig config, DataFlow::PathNode source, DataFlow::PathNode sink
 where config.hasFlowPath(source, sink)
-select sink.getNode(), source, sink, "Extraction of zipfile from $@", source.getNode(),
-  "a potentially untrusted source"
+select source.getNode(), source, sink,
+  "Unsanitized archive entry, which may contain '..', is used in a $@.", sink.getNode(),
+  "file system operation"
```

Comment:
This, to me, seems in want of an article at the beginning ("An unsanitized archive entry ... is used ..." or perhaps even "This ...".). If this is the way these messages are usually formulated, however, then I have no objection.

Reply:
I'll add a `This `.